src/main/java/duke/Event.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+
+/**
+ * Adds on with an at field which allows users to set an event timing.
+ *
+ * @author Yuvaraj Kumaresan
+ */
+public class Event extends Task {
+
+    /**
+     * Time at which the event is at.
+     */
+    protected String at;
```

Comment:
It's better to name it as eventTime or time. Others might be confused what "at" means.